keymanapp/keyman
# chore(common): Add crowdin strings for Czech

Adds crowdin strings for Czech for locale cs-CZ
Using localized name `čeština` from https://github.com/unicode-org/cldr/blob/a88558301989b98d5bedb13e9123ec6d12f82416/common/main/cs.xml#L142

## User Testing
This adds the crowdin strings for Kannada - locale kn-IN

TODO for @sgschantz 

## User Testing
Load the PR build for each platform and set the UI / Locale to "čeština" (Czech)

* **TEST_ANDROID** 
1. Load the PR build of Keyman for Android
2. In the Keyman app, go to Settings --> Display Language --> čeština (Czech)
3. Return to the Keyman app 
4. Verify the UI is in Czech (prompt, Settings strings, etc.).
![czech android](https://user-images.githubusercontent.com/7358010/214196149-f9454099-ca6f-4f87-9fb1-6463e571b849.jpg)



* **TEST_IOS**
1. Load the PR build of Keyman for iPhone and iPad
2. Set the device locale to Czech
3. Return to the Keyman app
4. Verify the UI is in  Czech

* **TEST_LINUX**
1. Start Keyman configuration with: LANGUAGE=cs_CZ km-config
2. Verify Keyman UI is in Kannada

* **TEST_MACOS**
1. Load the PR build of Keyman for macOS
2. Set the locale to Czech
3. Verify the Keyman UI is in Czech

* **TEST_WINDOWS**
1. Load the PR build of Keyman for Windows
2. Set the UI to čeština (Czech)
3. Verify the Keyman UI is in Czech
